HollowKnightStoredData.cs

```diff
@@ -1,139 +1,198 @@
-        public bool CheckChanged(Offset offset) {
-            bool ans = mem.PlayerData<int>(offset) != GetValue(offset);
-            return ans;
-        }
-        /// <summary>
-        /// Checks if the PD int given by offset has increased since the last update
-        /// </summary>
-        /// <param name="offset"></param>
-        /// <returns></returns>
-        public bool CheckIncreased(Offset offset) {
-            bool ans = mem.PlayerData<int>(offset) > GetValue(offset);
-            return ans;
-        }
-
-        /// <summary>
-        /// Checks if the PD bool given by offset has toggled since the last update
-        /// </summary>
-        /// <param name="offset"></param>
-        /// <returns></returns>
-        public bool CheckToggled(Offset offset) {
-            bool current = mem.PlayerData<bool>(offset);
-            bool previous = GetBoolValue(offset);
-            return current != previous;
-        }
-
-        /// <summary>
-        /// Checks if the PD bool given by offset has toggled from False to True since the last update
-        /// </summary>
-        /// <param name="offset"></param>
-        /// <returns></returns>
-        public bool CheckToggledTrue(Offset offset) {
-            bool current = mem.PlayerData<bool>(offset);
-            bool previous = GetBoolValue(offset);
-            return (current == true) && (previous == false);
-        }
-
-        /// <summary>
-        /// Checks if the PD bool given by offset has toggled from True to False since the last update
-        /// </summary>
-        /// <param name="offset"></param>
-        /// <returns></returns>
-        public bool CheckToggledFalse(Offset offset) {
-            bool current = mem.PlayerData<bool>(offset);
-            bool previous = GetBoolValue(offset);
-            return (current == false) && (previous == true);
-        }
-
-        public HollowKnightStoredData(HollowKnightMemory mem) {
-            this.mem = mem;
-        }
-
-        public void Update() {
-            if (CheckIncremented(Offset.dreamOrbs) && mem.SceneName() == "RestingGrounds_08") {
-                GladeEssence++;
-            }
-            foreach (Offset offset in pdInts.Keys) {
-                pdInts[offset] = mem.PlayerData<int>(offset);
-            }
-            foreach (Offset offset in pdBools.Keys) {
-                pdBools[offset] = mem.PlayerData<bool>(offset);
-            }
-
-            if (mem.HeroTransitionState() != HeroTransitionState.WAITING_TO_TRANSITION 
-                || mem.GameState() is GameState.EXITING_LEVEL or GameState.LOADING
-                || mem.SceneName() != mem.NextSceneName()) {
-                // In transition
-                TraitorLordDeadOnEntry = mem.PlayerData<bool>(Offset.killedTraitorLord);
-            } else {
-                // Not in transition
-                SplitThisTransition = false;
-            }
-        }
-    }
-}
+﻿using System.Collections.Concurrent;

```

Comment:
I think all the line endings changed in this file? I can hide whitespace to review the actual changes, but it would be nice not to need to